tests/Browser/LoginTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Tests\Browser;
+
+use Config;
+use Laravel\Dusk\Browser;
+use Tests\Authentication;
+use Tests\DuskTestCase;
+
+class LoginTest extends DuskTestCase {
+    /**
+     * Test the login sequence.
+     */
+    public function testLogin(): void {
+        Authentication::checkTestUser();
+        $this->browse(function (Browser $browser) {
+            $browser->visit('https://auth-dev.vatsim.net/login')
```

Comment:
Does this need to happen or should we direct to our login page and then verify that it goes to this page? This may be what's happening - I'm not too familiar with this

Reply:
I went back and forth on this. Does it *need* to happen? No. Is it a nice check to ensure that our dev login credentials still work? Yes. I tried to have this test the full login sequence but I couldn't find a way to do it because cloudflare's captcha gets in the way.

Reply:
This is not testing our website, at this point we are testing VATSIM's website - I don't think this should be included. We will know if our dev credentials stop working - it doesn't affect the production website in any way and this is more likely to get us all blocked by Cloudflare having this running constantly in the background

Reply:
And our dev credentials not working certainly should not prevent PRs from being merged

Reply:
Done - removed.